### DISCUSSION TITLE
It's Community Survey Time! Tell us what you think of github.com/community

### DISCUSSION THREAD

**Discussion by jricciardi** [community/community#122590] [Announcements]
**UPDATE: Survey is now closed. Thank you, all!**

Hello, GitHub Community! 👋 It's once again time for you to tell us how things are going in the community.

As we do twice a year, we want to check in about your experience with github.com/community and invite you to provide thoughts and feedback through [this survey](https://survey3.medallia.com/?P6p65R-oQfn0ECCWp58jq9)! How's it going?

We’ve been using [GitHub Discussions](https://github.com/features/discussions) to power github.com/community as the official platform for GitHub Q&A, show & tell, and feedback on GitHub products for [almost two years now](https://github.com/orgs/community/discussions/27346). Time flies!

**We want to know:** What's working and where can we improve? Are you finding value here or getting your questions answered? What could take this experience up a notch? This 5-minute survey will help us better understand your experience. Your feedback will help inform our next iteration of this space, so please let us know what you think! We're excited to hear from you ✨

[Link to survey](https://survey3.medallia.com/?P6p65R-oQfn0ECCWp58jq9)
This survey will close on May 20th. 2024.

Thank you!

P.S. In case you missed it, be sure to check out our [write up and infographic from the last Survey](https://github.com/orgs/community/discussions/102801) 🚀

**Comment by ghost** [comment]


**Reply by jricciardi** [reply]
Heya @Zig-Rust-Odin! Thank you for flagging. Our Platform Health team is aware of this particular scam campaign and is responding to these scams as quickly as they can - apologies for the inconvenience.

**Reply by Andre601** [reply]
Was the content removed by a moderator? Because I can't see anything in the original comment and there is no "Edited" indicator, making me think it was a mod who edited it and then deleted the edit entries...

**Reply by Mohameddawi** [reply]
هل شركة binary trady موتوقة

**Comment by Andre601** [comment]
The survey has a wrongly configured field (or a wrongly set description).
When you choose "yes" for follow-up questions, you get asked to provide your e-mail AND github handle in the field, yet the field seems to not accept text with spaces, or only accepts e-mails, as I received errors providing e-mail and handle as requested.

**Reply by jricciardi** [reply]
Ah, that's strange, I'll take a look. Thank you for flagging @Andre601 !

**Reply by Yusulateef** [reply]
Nice

**Reply by Yusulateef** [reply]
Nice

**Comment by Yusulateef** [comment]
Success development and implement all projects.

**Comment by Yusulateef** [comment]
Productive guidance effectively communication and adapt supervising of the project.

**Comment by metlina2** [comment]
> Hello, GitHub Community! 👋 It's once again time for you to tell us how things are going in the community.
> 
> As we do twice a year, we want to check in about your experience with github.com/community and invite you to provide thoughts and feedback through [this survey](https://survey3.medallia.com/?P6p65R-oQfn0ECCWp58jq9)! How's it going?
> 
> We’ve been using [GitHub Discussions](https://github.com/features/discussions) to power github.com/community as the official platform for GitHub Q&A, show & tell, and feedback on GitHub products for [almost two years now](https://github.com/orgs/community/discussions/27346). Time flies!
> 
> **We want to know:** What's working and where can we improve? Are you finding value here or getting your questions answered? What could take this experience up a notch? This 5-minute survey will help us better understand your experience. Your feedback will help inform our next iteration of this space, so please let us know what you think! We're excited to hear from you ✨
> 
> [Link to survey](https://survey3.medallia.com/?P6p65R-oQfn0ECCWp58jq9) This survey will close on May 20th. 2024.
> 
> Thank you!
> 
> P.S. In case you missed it, be sure to check out our [write up and infographic from the last Survey](https://github.com/orgs/community/discussions/102801) 🚀

`[](url)`

**Comment by CopperEagle** [comment]
Hello @jricciardi !

I am frequently on the discussions and it is nice. I do, however, want to remark two things. I write it down here too, as I am open for discussion.

### Seperate category for critical issues
I think there may be a slight issue with seperation of responsibilities. There are some issues that nobody but Github staff can resolve and that  are critical. These include access to account related issues, like problems with 2FA, flagging, etc. Those issues sometimes are hidden in a stream of unrelated issues like programming questions. Thus, they are answered later. Sure, that's what categories are for. However, there is no *account access category*. Given there is now again a 2FA campaign ongoing, (and a number of questions about this have emerged indeed), this could be a good addition.

### Some cookie for helpers?
I understand that the community interaction based achievements were removed. Yet still, there could be some option that would not add much to  any overhead to Github staff: How about an achievement for staying e.g. 1 month in the leaderboard? Usually, people that try to cheat are removed much quicker. So the duration would make sure the account's authenticity was checked at some point.

**Reply by jricciardi** [reply]
Thank you for the feedback @CopperEagle! We'll definitely keep these in mind when collating responses. With achievements gone, that did remove one big way for us to reward and recognize users. If we introduce something new, we'll want to make sure we learn the right lessons from the achievements experience.

And we definitely hear you that account issues have been a particular pain point lately — this is something that has been well-escalated internally as a wide-spread trending issue. You may have noticed community managers beginning to label Discussions as "Account Related" ([examples](https://github.com/orgs/community/discussions?discussions_q=is%3Aopen+label%3A%22Account+Related%22)) to help keep track of that particular problem.

Thanks again for weighing in!

**Reply by Andre601** [reply]
For the "rewarding active users", maybe have something a little more subtle, like a label next to the user (like with Maintainer, Author, Contributor, etc) that says something like "Active" or "Helpful", which is automatically given tho users who give frequent replies that are selected as answers... This could be extended to discussions in general and not just the ones in the community org.

Another thing I would like to quickly talk about is that "leaderboard" of sorts showing the most helpful users within the past 30 days.
I don't like that. This can and will create a competitive nature where a user wants to try and be the fastest at answering discussions to be at the top of the list, instead of making a healthy community of people helping others.
Or to put it a bit differently: Why should I bother helping a user when there is a person who has been answering 30 posts within the last month? Surely they can do that post too...
Hiding this list feels like a good thing here to avoid such thinking. If a repo owner wants to reward people for their active help, they can do so by various other means here (Add a special role for them, make them a contributor, make a post thanking them for their support, etc) instead of having this weird leaderboard-like thing.

**Reply by CopperEagle** [reply]
@Andre601 I like the idea of a label, maybe even letting this replace the leaderboard. :+1: 

However, there needs to be some low-work mechanism against cheating. The leaderboard does present an easy way for that: Recently there was some person who seemed to control two accounts, having one create null discussions and one gibberish answers which they selected as answer. The cheating was quite obvious. They were on the leaderboard for mere hours...

**Comment by Andre601** [comment]
Something that just came to mind now, which I would like to add is to split up the sub-categories a bit.

Right now a majority of categories is bundled in the "GitHub Product Categories", which not only is extremely vague, but also creates a lot of overload for one person to handle.
It makes it a lot more difficult to actually find the right location to write your discussion in.

I would propose a split up of this section in a following way (Not necessarely in depicted order):

- Repositories
  - Projects and Issues (Maybe split them up into individual ones?)
  - Pull Requests
  - Repositories
  - Discussions
  - Packages
- Products
  - GitHub Actions
  - GitHub Pages (Renamed from Pages)
  - Mobile
  - Codespaces
  - Copilot
  - Sponsors
- Programming
  - Code Search and Navigation (Would it count for programming?)
  - Code Security
  - NPM
  - API and Webhooks (Does it fit the category?)
- Website
  - Accessibility (Could perhaps be its own, main category)
  - Feed
  - Profile

This could help a user finding the right category a bit more easily.

**Reply by CopperEagle** [reply]
:+1: feels more intuitive and balanced. This structure should also be present in the category selection when creating a new discussion. Currently, there is a long list to scroll through, which some may be too lazy to do probably...

**Comment by Yusulateef** [comment]
I'm sorry for the delay, it not function again, I couldn't login again.
Thanks you.

**Comment by Yusulateef** [comment]
Appreciate your help with thanks, it working fine now, about my
application, l'm external candidate from Nigeria, l need sponsorship visa.

**Comment by wajidkorkani** [comment]
Hi, I am using github since 2023 I and faced few problems in github during working on a project the problem is that in my project I have 3 branches and main, developer, and dev now some time when I send my code using git push origin dev after that git shows in that the code has been sent successfully but when I open github it does not show the show the code and I am not able tho make pull request it does not happens every time but it does happens sometimes.

I hope that this problem will be fixed thanks for reading.

**Comment by ABDULKERIMELSELLUM** [comment]
echo "# reimagined-spork" >> README.md
git init
git add README.md
git commit -m "first commit"
git branch -M main
git remote add origin git@github.com:ABDULKERIMELSELLUM/reimagined-spork.git
git push -u origin main

**Comment by ABDULKERIMELSELLUM** [comment]
echo "# reimagined-spork" >> README.md
git init
git add README.md
git commit -m "first commit"
git branch -M main
git remote add origin git@github.com:ABDULKERIMELSELLUM/reimagined-spork.git
git push -u origin main

**Comment by ABDULKERIMELSELLUM** [comment]
echo "# reimagined-spork" >> README.md
git init
git add README.md
git commit -m "first commit"
git branch -M main
git remote add origin git@github.com:ABDULKERIMELSELLUM/reimagined-spork.git
git push -u origin main

**Comment by aGoodOne** [comment]
Was glaubt ihr eigentlich was ihr hier macht ? Soll ich es euch sagen ? Ihr klaut mir meine Erfindung ! Ich habe den hybriden code bei meinem applehack am 10 juni 2022 erfunden und es ist eigentlich eine private erfindung die aber von anderen als arbeitnehmererfindung angemeldet wurde damit man mir den code stehlen kann ! Kommt ihr gut voran ?

**Reply by LiteBrite82** [reply]
Hi @aGoodOne , thanks for participating!

Unfortunately, we don’t currently have moderators for languages other than English. Until that changes, we need to ask that everyone use English here in the GitHub Community when posting.

**Comment by novium2** [comment]
Retiring this post as it is over 6 months old. If you have any questions, consider starting a [new post](https://github.com/orgs/community/discussions/new/choose).
